docs/sources/investigate/flame-graph-ai.md

```diff
@@ -0,0 +1,91 @@
+
+A flame graph provides a convenient way to visualize performance data.
+These graphs provide a clear, intuitive understanding of resource allocation and bottlenecks within an application.
+To learn more, refer to [Flame graph visualizations](ref:flame-graph-panel).
+
+Flame graphs and profiling data in general can help you understand:
+
+1. Performance bottlenecks: What's causing the slowdown?
+1. Root causes: Why is it happening?
+1. Recommended fixes: How would you resolve it?
+
+Flame graph AI answers these questions when explaining your flame graph.
+To learn more, refer to [AI-powered insights for continuous profiling: introducing Flame graph AI in Grafana Cloud](https://grafana.com/blog/2024/05/15/ai-powered-insights-for-continuous-profiling-introducing-flame-graph-ai-in-grafana-cloud/).
+
+![Flame graph analysis with performance bottleneck, root cause, and recommended fix](/media/docs/grafana-cloud/profiles/pyorsope-flamegraph-ai-analysis.png)
+
+## Flame graph AI compliments human interpretation
+
+Flame graph AI's strength lies not just in analysis but more importantly in filling the gaps by augmenting and enhancing user experience.
+
+It only takes using a flame graph successfully once to really move from the beginner to advanced interpretation.
+From a product standpoint, one challenge has been building a user-experience that can span from beginner to expert and still be useful for both.
+
+When compared to volunteers in a limited flame graph interpretation test, the Flame graph AI's interpreter scored 100%. Beginners scored 25% and flame graph experts scored 83%.
+The AI consistently outperformed beginners and advanced users, providing accurate, albeit less detailed/nuanced, interpretations than the experts.
+
+These initial results at least point towards a great opportunity in adding value to most users by incorporating AI.
+
+Refer to the [AI-powered flame graph interpreter](https://pyroscope.io/blog/ai-powered-flamegraph-interpreter/) blog post to learn more.
+
+## Explore your profile data with Flame graph AI
+
+Flame graph AI uses the LLM plugin for Grafana to provide the large-language model using OpenAI API.
+
+You can use Flame graph AI in Grafana **Explore** > **Profiles** and the Explore Profiles app, nested underneath **Explore** > **Profiles** > **Explore profiles**.
+
+### Before you begin
+
+To use Flame graph AI, you must have:
+
+- Configured a [Grafana Pyroscope data source](https://grafana.com/docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/) that has profiling data
+
+- Enabled the [LLM plugin](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/configure/llm-plugin/) for your Grafana instance
+- Optional: Activated the [Explore Profiles](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/profiles/access/) app for your Grafana instance
```

Comment:
Do we have a way to use Flame graph AI without the Explore Profiles app @grafakus? I don't think so, so this step isn't optional.

Reply:
Is Flame Graph AI available in flame graph panels within Grafana dashboards?
```suggestion
- Activated the [Explore Profiles](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/profiles/access/) app for your Grafana instance
```

Reply:
Did some checking - Doesn't look like flame graph AI is available in the general flame graph panel on play.grafana.com but it is in the Explore Profiles UI.